bouncer/shared/lp_api_test.ts

```diff
@@ -70,6 +70,8 @@ async function provideLiquidityAndTestAssetBalances() {
 }
 
 async function testRegisterLiquidityRefundAddress() {
+  await using chainflip = await getChainflipApi();
```

Comment:
Why is this necessary when we have `chainflip` in the global scope?

Reply:
When using `await using` I think best not to put in the global scope, as it works like `Drop` in rust. Becomes much harder to reason about when it's in the global scope. So if there is a global usage, we should change that. 

Note that this function is doing something like an Arc.clone() and isn't re-initialising the whole client.

Reply:
Ah, I just realised that the global one was deleted. I was mostly saying that it didn't make sense to keep both. Must have looked at my local branch.